SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -533,9 +551,15 @@ void CNEMOEulerSolver::SetTime_Step(CGeometry *geometry, CSolver **solver_contai
 
         if (!geometry->nodes->GetDomain(iPoint)) continue;
 
-        Normal = geometry->vertex[iMarker][iVertex]->GetNormal();
-        Area = GeometryToolbox::Norm(nDim, Normal);
+        geometry->vertex[iMarker][iVertex]->GetNormal(Normal);
+
+        /*--- Set Normal to zero if node is in the Symmetry Plane marker ---*/
+        if(config->GetMarker_All_KindBC(iMarker) == SYMMETRY_PLANE) {          
+          for(iDim = 0; iDim < nDim; iDim++)
+            Normal[iDim]=0;
+        }
```

Comment:
Can't you skip the marker if it is a symmetry instead?

Reply:
Nicely spotted. Changing it